### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bennypowers**
Firefox already implemented, as mentioned on line 220

### CODE REVIEW

**Comment by littledan** [comment]
Firefox is still missing private fields, isn't it?

**Comment by bennypowers** [comment]
That's correct

File: README.md [hunk 1]
```diff
@@ -225,7 +225,6 @@ Further implementations are on the way:
 
 - [In progress](https://github.com/microsoft/TypeScript/pull/30829) in TypeScript
 - [Out for review](https://bugs.webkit.org/show_bug.cgi?id=174212) in Safari/JSC
-- [In progress](https://bugzilla.mozilla.org/show_bug.cgi?id=1499448) in Firefox/SpiderMonkey
```
Comment (legendecas) [inline README.md]: Maybe we can add a link above like others which refers to the version the feature released in?

**Comment by littledan** [comment]
So, deleting this line doesn't make sense, since this proposal includes private fields.

**Comment by bennypowers** [comment]
closing in light of https://github.com/tc39/proposal-class-fields/pull/292#issuecomment-619005553

**Pull request closed without merging**